### PR TITLE
fix: Updated docs with instructions for installing specific version #780

### PR DESCRIPTION
**Pull request by aemmadi**
**Summary**
Updated docs with example of using a specific version of docsify while getting started, instead of the default version. 

Issue #780 requested changes in the CLI, opened a PR there as well with the changes [#107](https://github.com/docsifyjs/docsify-cli/pull/107)

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [x] Chrome
- [x] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

---

* [x] DO NOT include files inside `lib` directory.

### CODE REVIEW

**Comment by aemmadi** [comment]
@jhildenbiddle Just pushed the requested changes. Also updated [docsify-cli#107](https://github.com/docsifyjs/docsify-cli/pull/107) accordingly.

**Comment by trusktr** [comment]
@aemmadi thanks for making this PR!

Besides having only one HTML block I think we should still explicitly mention in the wording outside of the block to highly recommend having a version number.

@aemmadi mind making one more change? to add a note about using the version so that updates "don't break your website"?

**Comment by jhildenbiddle** [comment]
Good point, @trusktr.

A description of why docsify URLs are locked to a specific version (`@4`) will be helpful, along with an explanation that URLs will need to be manually updated re receive new major versions. Perhaps presenting sample URL schemes would help illustrate:

---

#### Specifying docsify versions

Specifying a major version in the URL (`@4`) will allow your site will receive non-breaking enhancements (i.e. "minor" updates) and bug fixes (i.e. "patch" updates) automatically. This is the recommended way to load docsify resources.

```md
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/themes/vue.css">
<script src="//cdn.jsdelivr.net/npm/docsify@4/lib/docsify.min.js"></script>
```

If you prefer to lock docsify to a specific version, specify the full version after the `@` symbol in the URL. This is the safest way to ensure your site will look and behave the same way regardless of any changes made to future versions of docsify.

```md
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4.11.4/themes/vue.css">
<script src="//cdn.jsdelivr.net/npm/docsify@4.11.4/lib/docsify.min.js"></script>
```

?> Note that in both of the examples above, docsify URLs will need to be manually updated when a new major version of docsify is released (e.g. v4.x.x => v5.x.x). Check the docsify website periodically to see if a new major version has been released.

---

Thoughts? Too wordy?

**Comment by aemmadi** [comment]
> ```lisp
> <script src="//cdn.jsdelivr.net/npm/docsify@4/lib/docsify.min.js"></script>
> ```

> ```lisp
> <script src="//cdn.jsdelivr.net/npm/docsify@4.11.4/lib/docsify.min.js"></script>
> ```

I noticed you used the full file path in `jsdelivr`, you mentioned in [previous comment](https://github.com/docsifyjs/docsify/pull/1225#pullrequestreview-432662081) to remove any file names and paths since `jsdelivr` takes care of that. 

So the `src` for the `<script>` tag earlier would be:
```html
<script src="//cdn.jsdelivr.net/npm/docsify@4"></script>
<script src="//cdn.jsdelivr.net/npm/docsify@4.11.4"></script>
```

**Comment by jhildenbiddle** [comment]
@aemmadi --

Good catch. Sorry about that. Copy/Paste error. I've updated my comment by removing the full path to `docsify.min.js`.

**Comment by aemmadi** [comment]
I added all the requested changes and made sub headings to make the doc look aesthetic. Let me know if that works!

**Comment by anikethsaha** [comment]
cc @jhildenbiddle can you review this as there is one change request from you. 

@aemmadi can you rebase with the upstream . 

Thanks

File: docs/quickstart.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ docsify serve docs
 
 ## Manual initialization
 
-If you don't like `npm` or have trouble installing the tool, you can manually create `index.html`:
+If you don't like `npm` or have trouble installing the tool, you can manually create `index.html` (the code below will load the latest version of docsify automatically. If you want a specific version to avoid any broken elements in the future, [see below](#specifying-docsify-versions)):
 
 ```html
 <!-- index.html -->
```
Comment (jhildenbiddle) [inline docs/quickstart.md:42]: Both the `<link>` and `<script src="...">` elements should be locked to v4 in this HTML block.

```html
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/themes/vue.css">
```

```html
<script src="//cdn.jsdelivr.net/npm/docsify@4"></script>
```

File: docs/quickstart.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ docsify serve docs
 
 ## Manual initialization
 
-If you don't like `npm` or have trouble installing the tool, you can manually create `index.html`:
+If you don't like `npm` or have trouble installing the tool, you can manually create `index.html` (the code below will load the latest version of docsify automatically. If you want a specific version to avoid any broken elements in the future, [see below](#specifying-docsify-versions)):
```
Comment (jhildenbiddle) [inline docs/quickstart.md:39]: You can revert this change.

We do not want people to load the latest version of docsify automatically. We want them to get the latest minor and patch releases, but have to manually update major releases. This will prevent their site from breaking when a new major version is released.

File: docs/quickstart.md [hunk 3]
```diff
@@ -61,6 +61,26 @@ If you don't like `npm` or have trouble installing the tool, you can manually cr
 </html>
 ```
 
+### Specifying docsify versions
```
Comment (jhildenbiddle) [inline docs/quickstart.md:64]: Insert above this header, under the HTML block:

Specifying a major version in the URL (@4) will allow your site will receive non-breaking enhancements (i.e. "minor" updates) and bug fixes (i.e. "patch" updates) automatically. This is the recommended way to load docsify resources.

File: docs/quickstart.md (hunk 1 above)
Comment (aemmadi) [reply docs/quickstart.md:42]: Oh, I assumed we still wanted the latest version block. No worries, will edit that asap!

**Comment by aemmadi** [comment]
@jhildenbiddle Pushed the changes!

**Comment by anikethsaha** [comment]
idk whats wrong but I cant able to update the branch, @aemmadi can you update  your branch .

**Comment by aemmadi** [comment]
> idk whats wrong but I cant able to update the branch, @aemmadi can you update your branch .

Weird, I updated it to the `develop` branch
